mteb/load_results/task_results.py

```diff
@@ -511,12 +511,14 @@ def validate_and_filter_scores(self, task: AbsTask | None = None) -> AbsTask:
                 new_scores[split].append(_scores)
                 seen_subsets.add(_scores["hf_subset"])
             if seen_subsets != hf_subsets:
-                raise ValueError(
-                    f"Missing subsets {hf_subsets - seen_subsets} for split {split}"
+                logger.warning(
+                    f"{task.metadata.name}: Missing subsets {hf_subsets - seen_subsets} for split {split}"
                 )
             seen_splits.add(split)
         if seen_splits != set(splits):
-            raise ValueError(f"Missing splits {set(splits) - seen_splits}")
+            logger.warning(
+                f"{task.metadata.name}: Missing splits {set(splits) - seen_splits}"
+            )
```

Comment:
This seems problematic to turn into a warning -  then it would get added to the leaderboard even though not all the results are there?

Reply:
It wouldn't because it fails at the get_score step. This just allows all splits to be loaded, which is beneficial when you have a benchmark, that only uses that split. (I hope I'm making sense)

Reply:
So e.g. `results` repo only has results on the `test` split, but the task description says `eval_splits=["dev", "test"]`. If you throw an error instead of a warning, you don't load the task results, so it will be missing, even if you have a benchmark that only has `eval_splits=["test"]`

Reply:
check!